specs/windows/winbaseobj.table

```diff
@@ -1,5 +1,5 @@
 table_name("winbaseobj")
-description("Lists named Windows objects in the default object directories, across all terminal services sessions.")
```

Comment:
Tests are failing due to linting here, do you mind adding a `.` to the end of the description?

Reply:
done!